frog/imports/ui/Analytics.jsx

```diff
@@ -0,0 +1,9 @@
+import React, { Component } from 'react';
+ 
+export default class Analytics extends Component {
+  render() {
+    return (
+      <p>You are on the Analytics app</p>
+    );
+  }
+}
```

Comment:
All the more reason to keep it simple and separate concerns... One thing we need to talk about is how to deal with internal state in the app. I need to read a bit more about how people in Meteor do that - I have good experiences with Redux and saw some blog posts about using Meteor with Redux, but need to read them before I recommend it :)